portfolio/src/main/java/com/google/sps/servlets/LoadFormServlet.java

```diff
@@ -0,0 +1,54 @@
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.gson.Gson;
+
+/** Servlet that processes text. */
+@WebServlet("/load-form")
+public final class LoadFormServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+
+        // used to load data
+        Query<Entity> query = Query.newEntityQueryBuilder().setKind("Form").setOrderBy(OrderBy.desc("timestamp"))
+                .build();
+        QueryResults<Entity> results = datastore.run(query);
+
+        ArrayList<Object> forms = new ArrayList<>();
```

Comment:
Just noticed this. Seeing `Object` is generally a bad sign. If you make `content` an `ArrayList<String>`, then this can be

```suggestion
        ArrayList<ArrayList<String>> forms = new ArrayList<>();
```